uint16 asf_file
---
# Success if empty
string error

